lib/maths/CSeasonalComponentAdaptiveBucketing.cc

```diff
@@ -333,7 +333,9 @@ bool CSeasonalComponentAdaptiveBucketing::acceptRestoreTraverser(core::CStateRes
             RESTORE(LAST_UPDATES_OLD_TAG,
                     core::CPersistUtils::fromString(traverser.value(), lastUpdates))
         } while (traverser.next());
-
+        if (lastUpdates.empty()) {
```

Comment:
Could add a comment to say this is needed because `lastUpdates` wasn't present in 5.6.